aspnetcore/performance/rate-limit.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Rate limiting middleware in ASP.NET Core
+author: rick-anderson
+description: Learn how limit requests in ASP.NET Core apps
+ms.author: riande
+ms.custom: mvc
+ms.date: 8/28/2022
+uid: performance/rate-limit
+---
+
+# Rate limiting middleware in ASP.NET Core
+
+By [Arvin Kahbazi](https://github.com/Kahbazi) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+:::moniker range=">= aspnetcore-7.0"
+
+The [Microsoft.AspNetCore.RateLimiting](https://www.nuget.org/packages/Microsoft.AspNetCore.RateLimiting) NuGet package provides rate limiting middleware. Apps configure rate limiting policies and then attach the policies to endpoints. Apps using rate limiting should be carefully load tested before deploying. See [Testing endpoints with rate limiting](#test7) in this article for more information.
+
+## Rate limiter algorithms
+
+The [`RateLimiterOptionsExtensions`](/dotnet/api/microsoft.aspnetcore.ratelimiting.ratelimiteroptionsextensions) class provides the following extension methods for rate limiting:
+
+* [Fixed window](#fixed)
+* [Sliding window](#slide)
+* [Token bucket](#token)
+* [Concurrency](#concur)
+
+<a name="fixed"></a>
+
+### Fixed window limiter
+
+The [`AddFixedWindowLimiter`](/dotnet/api/microsoft.aspnetcore.ratelimiting.ratelimiteroptionsextensions.addfixedwindowlimiter#microsoft-aspnetcore-ratelimiting-ratelimiteroptionsextensions-addfixedwindowlimiter(microsoft-aspnetcore-ratelimiting-ratelimiteroptions-system-string-system-threading-ratelimiting-fixedwindowratelimiteroptions)) uses a fixed time window to limit requests. When the time window expires, a new time window starts and the request limit is reset.
+
+Consider the following code:
+
+:::code language="csharp" source="~/../AspNetCore.Docs.Samples/fundamentals/middleware/rate-limit/WebRateLimitAuth/Program.cs" id="snippet_fixed":::
+
+The preceding code:
+* Calls [UseRateLimiter](/dotnet/api/microsoft.aspnetcore.builder.ratelimiterapplicationbuilderextensions.useratelimiter) to enable rate limiting.
```

Comment:
This link appears to be broken in the local copy I'm looking at.

![image](https://user-images.githubusercontent.com/1439341/186719351-25dc187c-48b8-47c9-9008-cdc626d16f11.png)

Reply:
You have to remove `review.` from the URL
https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.ratelimiterapplicationbuilderextensions.useratelimiter?branch=pr-en-us-26831